README.md

```diff
@@ -41,15 +38,17 @@ the Checked C implementation has not broken existing tests.
 Note: These steps have been adopted from the [LNT Quickstart Guide](http://llvm.org/docs/lnt/quickstart.html).
 ```
 sudo apt-get install bison flex
-sudo apt-get install python-pip
-sudo pip install setuptools
-sudo easy_install virtualenv
-virtualenv ~/mysandbox
-svn co http://llvm.org/svn/llvm-project/lnt/trunk ~/lnt
-~/mysandbox/bin/python ~/lnt/setup.py install
+sudo apt-get install python3-pip
+sudo pip3 install setuptools
+sudo apt-get install virtualenv
+sudo apt-get install tclsh
+sudo virtualenv ~/mysandbox
+git clone https://github.com/llvm/llvm-lnt.git  ~/lnt
+sudo ~/mysandbox/bin/python ~/lnt/setup.py install
 ```
 
 2. Invoke LNT tests
+
```

Comment:
Extra newline.

Reply:
This is intentional - I wanted the prereqs to appear on a separate line.